src/core.jl

```diff
@@ -53,17 +53,37 @@ struct CircleThreePoints <: Circle
 end
 
 """
-    circle = CirclePointRadius(center, ρ)
+    circle = CirclePointRadius(center, ρ; thickness, fill)
 
 A `Drawable` circle having center `center` and radius `ρ`
+with keyword arguments `thickness` and `fill` for drawing hollow circle
+
+# Arguments
+- `thickness::Integer`: thickness of the circle 
```

Comment:
If you declare `thickness::Integer` is supported, then you need to add such tests for them, e.g., to make sure that `thickness=UInt8(1)` works.

Reply:
Also, the argument explanation should exist in docs of other structs, too. You won't expect the user of `Ellipse` to query the meaning of `thickness` by checking `?CirclePointRadius`.

Reply:
Sure,I'll fix all those issues..can you check the way thickness is to be implemented in a ellipse,since our circle draw is dependent on ellipse(otherwise it would have quite straightforward) and the thickness of elliptic curve is a difficult topic with parallel curves to be drawn

Reply:
I'm quite busy learning halide and the C++ binding toolchain recently 😢  I'll check it this weekend if you haven't figured it out then.

Reply:
Other than this one,I have added fixes for others .Inner and outer constructor got me quite confused especially with keyword arguments since the some test cases were able to see the inner constructor or maybe I missed something 😓 

> If you declare `thickness::Integer` is supported, then you need to add such tests for them, e.g., to make sure that `thickness=UInt8(1)` works.

Reply:
```suggestion
- `thickness::Int`: thickness of the circle
```